region/brinstar/pink.json

```diff
@@ -1687,12 +1952,108 @@
                   "note": "Climb up 1 screen."
                 }
               ]
+            },
+            {
+              "id": 15,
+              "strats": [
+                {
+                  "name": "G-mode Morph",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [8],
+                      "mode": "any",
+                      "artificialMorph": true
+                    }},
+                    {"or": [
+                      "h_canArtificialMorphIBJ",
+                      "Morph",
+                      {"and": [
+                        "SpringBall",
```

Comment:
Seems hard to get past the Reo at 6

Reply:
Coming in from 8, the two Reos below the hopper pit spawn far to the left, so they shouldnt be a problem here

Reply:
Jumping over the pipe looks to make the reo move left.  And that is easy enough to do with springball.